src/sudo/mod.rs

```diff
@@ -107,6 +104,7 @@ fn sudo_process() -> Result<(), Error> {
                     eprintln_ignore_io_error!("{}", help::USAGE_MSG);
                     std::process::exit(1);
                 } else {
+                    #[cfg(feature = "dev")]
```

Comment:
```suggestion
```

Reply:
I have a different idea which makes everything much simpler still.

 `#[allow(dead_code)]` and then keep this `#[cfg]` line.